sopel-modules/rat-board.py

```diff
@@ -690,8 +690,8 @@ def append_quotes(bot, search, lines, autocorrect=True, create=True, detect_plat
 
     json_lines = []
     for line in rv.added_lines:
-        json_lines.append({"message":line, "updatedAt":datetime.datetime.utcnow().isoformat(),
-                           "createdAt":datetime.datetime.utcnow().isoformat(), "author":author, "lastAuthor":author})
+        json_lines.append({"message":line, "updatedAt":datetime.datetime.now(datetime.timezone.utc).isoformat().replace('+00:00', 'Z'),
```

Comment:
does this actually work? @xlexi previously said there was "no timezone information" on the given Timestamp - that'd mean this replace wouldn't work. Will see tho if this works tomo--- tonight (~20h from now, heading to bed now)

Reply:
Yes because we're defining the timezone by specifying it when calling now(). The problem is that utcnow() will create a utc datetime object that does not contain tz info. Why it does this is beyond me, but it even notes that in the documentation.